lms/djangoapps/shoppingcart/tests/test_reports.py

```diff
@@ -108,11 +109,11 @@ def setUp(self, cutoff_date):
         second_refund.refund_requested_time = self.test_time
         second_refund.save()
 
-        self.CORRECT_REFUND_REPORT_CSV = dedent(u"""
+        self.CORRECT_REFUND_REPORT_CSV = dedent("""
```

Comment:
In python 2 this will make this a byte string instead of a unicod estring.  I think you want to keep the prefix `u` but not the suffix `.encode('utf-8')`.

Reply:
ok.